www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
+
+To update your Launchers, run:
+
+```shell
+hab pkg install core/hab-launcher
+```
+
+Then restart the Supervisor. This will, by necessity, require a restart of supervised services, so factor that into your planning.
+
+The Launcher is designed to run as process 1; it is minimal by design. Its responsibilities are to:
+
+* Launch the Habitat Supervisor
+* Restart the Habitat Supervisor on shutdown (Supervisor for the Supervisor)
+* Start, stop, and restart processes on behalf of the Supervisor
+
+The Launcher enables the Supervisor to update itself without shutting down or re-parenting the services that the Supervisor is supervising. The Launcher is versioned separately from the Supervisor and should be updated very infrequently since an update of the Launcher could require a system restart if it is running as process 1.
+
+The Laucher *always* starts the Supervisor. The Launcher always attempts to start the latest version of the Supervisor package installed on the host. When the Launcher starts the Supervisor it makes a handshake over a platform specific [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) channel. The Supervisor communicates with the Launcher with a binary protocol through the Launcher client.
+
+When the Supervisor needs to load and execute a new child process it sends a `spawn` message to the Launcher. The Launcher returns the process identification (PID) of the running process to the Supervisor if the child process launches successfully, and it returns a failure message if it does not. or a failure message.The Supervisor watches the spawned child process and asks the Launcher to stop or restart it, as needed. If the Supervisor crashes or is restarted by an update, it then re-attaches itself to all running services and continues normal operation.
```

Comment:
I'm not sure we necessarily need these last two paragraphs. The information is true, but seems to be very deep into the implementation for general consumption purposes.